contributions/attestation-data-duplicator.md

```diff
@@ -3,16 +3,16 @@ title: "Attestation Data Duplicator"
 description: "One of the primary challenges currently facing the AttestationStation is the need to bootstrap the system with high-quality attestations. Luckily, blockchains already have an enormous amount of useful data that can be converted into attestations."
 lang: "en-US"
 type: "Project Idea"
-authors: ["@smartcontracts (GitHub) @ OP Labs"]
+authors: ["@aagbotemi (GitHub) @ Duplikaat Team","@goodness5 (GitHub) @ Duplikaat Team","@okolievans (GitHub) @ Duplikaat Team","@olorunfemibabs (GitHub) @ Duplikaat Team", "@dadsec-dev (GitHub) @ Duplikaat Team"]
```

Comment:
Hi! Y'all are not the author, please remove this change.

Reply:
Hi @Nicca42, the change has been updated.